src/main/java/org/ibp/api/java/germplasm/GermplasmListService.java

```diff
@@ -32,6 +32,8 @@
 
 	List<MyListsDTO> getMyLists(String programUUID, Pageable pageable, Integer userId);
 
+	GermplasmListGeneratorDTO clone(Integer germplasmListId, GermplasmListDto request);
```

Comment:
Maybe return GermplasmListDto instead unless the entries/data are being updated as well in Middleware after cloning